src/styles/ui-playground/ui-playground.scss

```diff
@@ -8,45 +44,114 @@ $background-color: #ededed;
 
 // Component styles
```

Comment:
I think it would be good to split these out into separate modules in future, similar to how we do for the `sidebar/` and `annotator/` directories within `src/styles/`